### ISSUE TITLE
recursive check broken: files outside of the current folder are not found

### ISSUE THREAD

**Issue by sbraz** [rhash/RHash#106]
Hi, thanks for fixing #104, however I've just noticed a new problem:
```
$ cat folder/subfolder/test.sfv 
; Generated by RHash v1.3.9 on 2020-01-03 at 00:14.05
; Written by Kravchenko Aleksey (Akademgorodok) - http://rhash.sf.net/
;
;            5  00:14.01 2020-01-03 test
test 3BB935C6
$ cat folder/subfolder/test
test
$ ./rhash -rc folder/

--( Verifying folder/subfolder/test.sfv )---------------------------------------
test                                                No such file or directory
--------------------------------------------------------------------------------
Errors Occurred: Errors:0   Miss:1   Success:0   Total:1  
```

Before c992231b3fba87e12346526ca2bd7efa07657353, files paths were checked relatively to the checksum file directory and this worked as expected.

**Comment by rhash** [comment]
Thanks for the bug report. This problem is result of two bugs, one of which resides within get_modified_path()/get_modified_tpath() functions.
I hope to find time to fix it within several next days.

**Comment by rhash** [comment]
The problem is fixed by commits: 5c0af713b86aea3fd3b9f6397381dbe0efe1cefd, 1e2990f983ba0f0ced599958e5ed12a79a35b4b5, 35f6f7f40ebd3186cc7ae4690f8a7c7b66b31d73.

Behavior of 'rhash -rc' is now restored to be the same of v1.3.8. An auto test for '-rc' is added. Also two memory problems were fixed. All fixes will be included into the next release.

**Comment by sbraz** [comment]
Thanks, what is the ETA for the new release? I'm wondering if I should port those patches to our Gentoo ebuild.

**Comment by rhash** [comment]
@sbraz you should port the patches. The next release is expected around the fall of this year.

**Comment by sbraz** [comment]
@rhash I'll do that then.
I don't know how you usually handle releases but almost a year seems a bit long, especially when one of the problem leads to a segfault. Would you please consider doing a bugfix release?

**Comment by rhash** [comment]
The fix is included into **RHash v1.4.0**.
